components/date-picker/locale/nb_NO.tsx

```diff
@@ -1,17 +1,53 @@
-import CalendarLocale from 'rc-picker/lib/locale/nb_NO';
```

Comment:
Why did you remove this?　You need to send PR to rc-picker　at　first

Reply:
Thanks for the quick reply. Create a pull there, https://github.com/react-component/picker/pull/97

Will update diff when/if thats merged and released

Reply:
Waiting a new patch.

Reply:
rc-picker v1.10.6, Pls remove `node_modules` and `package-lock.json` or `yarn.lock` then reinstall it.